### PR TITLE
tidb-in-kubernetes: make deploy doc clearer on GCP

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? 
Update deploy doc on GCP to make it clearer

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 1]
```diff
@@ -62,30 +62,41 @@ gcloud services enable cloudresourcemanager.googleapis.com \
 + `GCP_REGION`：创建资源所在的区域，例如：`us-west1`。
 + `GCP_PROJECT`：GCP 项目的名称。
 
-要使用上述 3 个环境变量配置 Terraform，可执行以下脚本：
+要使用上述 3 个环境变量配置 Terraform，可执行以下步骤：
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
若要使用以上 3 个环境变量来配置 Terraform，可执行以下步骤：
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 2]
```diff
@@ -62,30 +62,41 @@ gcloud services enable cloudresourcemanager.googleapis.com \
...
+
+将 GCP_REGION 替换为你的 GCP Region。
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
1. 将 `GCP_REGION` 替换为你的 GCP Region。
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 3]
```diff
@@ -62,30 +62,41 @@ gcloud services enable cloudresourcemanager.googleapis.com \
...
 
 {{< copyable "shell-regular" >}}
 
 ```bash
-# 将 GCP_REGION 替换为你的 GCP Region。
 echo GCP_REGION=\"us-west1\" >> terraform.tfvars
 ```
 
+将 GCP_PROJECT 替换为你的 GCP 项目名称，确保连接的是正确的 GCP 项目。
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
2. 将 `GCP_PROJECT` 替换为你的 GCP 项目名称，确保连接的是正确的 GCP 项目。
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 4]
```diff
@@ -62,30 +62,41 @@ gcloud services enable cloudresourcemanager.googleapis.com \
...
+
 {{< copyable "shell-regular" >}}
 
 ```bash
-# 将 GCP_PROJECT 替换为你的 GCP 项目名称，确保连接的是正确的 GCP 项目。
 echo "GCP_PROJECT=\"$(gcloud config get-value project)\"" >> terraform.tfvars
 ```
 
+初始化 Terraform：
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
3. 初始化 Terraform：
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 5]
```diff
@@ -62,30 +62,41 @@ gcloud services enable cloudresourcemanager.googleapis.com \
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+terraform init
+```
+
+为 Terraform 创建一个有限权限的服务账号，并设置证书路径。
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
4. 为 Terraform 创建一个有限权限的服务账号，并设置证书路径。
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 6]
```diff
@@ -349,21 +359,46 @@ GCP 允许 `n1-standard-1` 或者更大的实例类型挂载本地 SSD，这提
 
 ### 自定义 TiDB 参数配置
 
-Terraform 脚本为 GKE 中的 TiDB 集群提供了默认设置。你也可以在 `tidbclusters.tf` 中为每个 TiDB 集群指定一个覆盖文件 - `values.yaml`。此文件中的值会覆盖默认设置。
+Terraform 脚本为 GKE 中的 TiDB 集群提供了默认设置。你也可以在 `tidbclusters.tf` 中为每个 TiDB 集群指定一个覆盖文件 - `override_values`。此文件中的值会覆盖默认设置。
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
Terraform 脚本为 GKE 中的 TiDB 集群提供了默认设置。你也可以在 `tidbclusters.tf` 中为每个 TiDB 集群指定一个覆盖文件 `override_values`。此文件中的值会覆盖默认设置。
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 7]
```diff
@@ -349,21 +359,46 @@ GCP 允许 `n1-standard-1` 或者更大的实例类型挂载本地 SSD，这提
...
+
+例如，集群默认使用 `deploy/modules/gcp/tidb-cluster` 模块中的 `values/default.yaml` 作为覆盖文件。
 
-例如，集群默认使用 `gcp/tidb-cluster` 模块中的 `default.yaml` 作为覆盖文件，并在此文件中启用了 ConfigMap rollout 功能。
+在 GKE 中，某些值不支持在 `values.yaml` 中自定义，包括集群版本、副本数、`NodeSelector` 以及 `Tolerations`。`NodeSelector` 和 `Tolerations` 由 Terraform 直接管理以确保基础设施与 TiDB 集群之间的一致性。集群版本和副本数可以通过 `tidbclusters.tf` 文件中编辑每个 `tidb-cluster` module 参数来修改。
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
在 GKE 中，某些值不支持在 `values.yaml` 中自定义，包括集群版本、副本数、`NodeSelector` 以及 `Tolerations`。`NodeSelector` 和 `Tolerations` 由 Terraform 直接管理以确保基础设施与 TiDB 集群之间的一致性。若要自定义集群版本和副本数，可以修改 `tidbclusters.tf` 文件中编辑每个 `tidb-cluster` module 的参数。
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 8]
```diff
@@ -349,21 +359,46 @@ GCP 允许 `n1-standard-1` 或者更大的实例类型挂载本地 SSD，这提
...
 
-在 GKE 中，某些值不支持在 `values.yaml` 中自定义，例如群集版本、副本、节点选择器和 taints。这些变量由 Terraform 控制，以确保基础架构和 TiDB 集群一致。如果要自定义这些变量，可以直接在 `tidbclusters.tf` 文件中编辑每个 `tidb-cluster` 模块的变量。
+> **注意：**
+>
+> 自定义 `values.yaml` 配置文件中，不建议包含如下配置（`tidb-cluster` module 默认固定配置）：
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
> 自定义 `values.yaml` 配置文件中，不建议包含以下配置（`tidb-cluster` module 默认固定配置）：
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 9]
```diff
@@ -483,11 +518,12 @@ terraform destroy
 
 如果采用了最佳实践，集群中的 Terraform 状态不会相互干扰，并且扩展起来很方便。示例如下：
 
+假设已在项目根目录：
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]: `示例如下（假设已在项目根目录）：` 会不会好一些？

Comment (DanielZhangQD) [reply dev/tidb-in-kubernetes/deploy/gcp-gke.md]: ok

Comment (DanielZhangQD) [reply dev/tidb-in-kubernetes/deploy/gcp-gke.md]: ok

**Comment by DanielZhangQD** [comment]
Please do not merge this PR until https://github.com/pingcap/tidb-operator/pull/959 is merged.

**Comment by DanielZhangQD** [comment]
@TomShawn comments addressed, PTAL again.

**Comment by TomShawn** [comment]
LGTM

**Pull request merged**